workshop/cleanup/nuke_the_workshop.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+echo ''
+echo 'You are about to remove EVERYTHING on your Kubernetes cluster, including Istio.'
+echo ''
+echo 'You must put in Y as your answer.'
+read -p "Are you sure? " -n 1 -r
+echo
+if [[ $REPLY =~ ^[Y]$ ]]
+then
+  kubectl delete pods --all
```

Comment:
This scares me a little. Can't we just delete the resource we created by name?  If we do this, why not just tell them to delete the entire cluster - it would be easier.

Reply:
I was attempting to resolve https://github.com/IBM/istio101/issues/110 by @linsun.

I am going to ping her directly today about it to get her feedback.

Reply:
What if instead of deleting all resources we just run `kubectl delete -f` on each file we processed?